angular/projects/spark-angular/src/lib/components/sprk-dropdown/sprk-dropdown.component.ts

```diff
@@ -60,7 +62,7 @@ import { ISprkDropdownChoice } from './sprk-dropdown.interfaces';
             sprkLink
             *ngIf="selector && (!title || !heading)"
             variant="plain"
-            class="sprk-o-Stack sprk-o-Stack--split@xxs sprk-o-Stack--center-column sprk-u-Width-100"
```

Comment:
The `sprk-u-Width-100` did not need to be there because Stack applies width: 100